### PR TITLE
partly Fixes #134:Added skill for converting binary into decimal

### LINKED ISSUE #134: Conversion skills for susi
As suggestes by @hpdang I need to focus on susi conversion skills for Milestone 6

**Would you like to work on the issue?**

I will make sure that susi replies to all conversion type questions.
This issue is likely to recieve multiple PR's.
Thanks

### PR DESCRIPTION
**Pull request by madhavrathi**
partly fixes issue #134 

Changes: Added skill for binary to decimal conversion

Etherpad link: http://dream.susi.ai/p/binary_to_decimal
Screenshots for the change: 
hy

### CODE REVIEW

File: models/general/problemsolving/en/binaryToDecimal.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#The following code returns binary equivalent of a decimal number given by user
+
+convert * into decimal | decimal of *
```
Comment (Manveer695) [inline models/general/problemsolving/en/binaryToDecimal.txt]: can you add a question like - convert * to decimal. Thanks!

**Review by hpdang — CHANGES_REQUESTED** [review]
@madhavrathi please make sure, there's always at least 2 approved reviews. And could you please add this into your previous skill, we don't need two different skills for this https://github.com/fossasia/susi_skill_data/blob/master/models/general/problemsolving/en/decimalToBinary.txt

**Comment by madhavrathi** [comment]
@hpdang Made the required changes.
@dynamitechetan @saurabhjn76 @Manveer695 please review this PR.
Thanks

**Review by Laughing-Boy — APPROVED** [review]

**Comment by Laughing-Boy** [comment]
@madhavrathi ,Please add more questions

**Comment by madhavrathi** [comment]
@jyoth1raditya Please review !
Thanks

**Pull request merged**